django/contrib/sessions/backends/base.py

```diff
@@ -269,9 +275,9 @@ def get_expire_at_browser_close(self):
         ``get_expiry_date()`` or ``get_expiry_age()`` to find the actual expiry
         date/age, if there is one.
         """
-        if self.get('_session_expiry') is None:
+        if (expiry := self.get('_session_expiry')) is None:
```

Comment:
Is Python 3.7 still supported? If 3.7 is still supported, then walrus operator won't work.

Reply:
Hi Andrew,

> Is Python 3.7 still supported?

Django 4.0+ (which this would be targeting) has a minimum version requirement of python `3.8` ([see preliminary release notes](https://docs.djangoproject.com/en/dev/releases/4.0/#python-compatibility)), and the walrus operator is already in use elsewhere in the codebase, so on that front we should be fine.